### ISSUE TITLE
Adrian's comments

### ISSUE THREAD

**Issue by bsesar** [bsesar/Ophiuchus-stream#2]
Also, I was reading your Ophiuchus paper and noticed the bottom of page 10, you cite Sanders & Binney (2013) for noting the velocity dispersion scales with the cube root of the dynamical mass. This is actually a well known result that goes back at least to some of Kathryn's papers from the 90's. I think a more relevant citation may be: 
http://adsabs.harvard.edu/abs/1999ApJ...512L.109J

Note that we also show this from running N-body simulations over a range of masses in our 2014 paper:
http://adsabs.harvard.edu/abs/2014ApJ...794....4P

**Comment by jobovy** [comment]
Yes, this is a well-known result, but sometimes referees want a reference. I think we cited Sanders & Binney because that was what I cited in my stream paper. We could also add Johnston, K. V. 1998, ApJ, 495, 297.
